api/v1/swagger.yaml

```diff
@@ -1790,6 +1790,586 @@ paths:
           $ref: "#/responses/genericError"
 
   # ---------------------------------------------------------------------------
+
+  /generators:
+    get:
+      security:
+        - jwt: []
+      summary: Get a generator or all generators.
+      tags: [ generators ]
+      description: >-
+        Get a generator or all generators. Find a lens or lenses. You may query using field filters with asterisk
+        (*) wildcards. You may also optionally specify sort, limit, offset, and
+        a list of fields to include in the response. If the Refocus configuration parameter `useAccessToken` is set to `true`, you must include an `Authorization` request header with your [JSON Web Token](https://tools.ietf.org/html/rfc7519) (JWT) as the value. You can get a token using `POST /v1/register` or `POST /v1/tokens`.
+      externalDocs:
+        description: >-
+          GET /generators
+        url: http://focus.salesforce.com/docs/api/generators
+      operationId: findGenerators
+      parameters:
+        -
+          $ref: "#/parameters/GeneratorFieldsParam"
+        - name: sort
+          in: query
+          description: >
+            Specify the sort order using a field name, e.g. '...?sort=name'. Prepend the
+            field name with a minus sign to specify descending order, e.g.
+            '...?sort=-name'.
+          required: false
+          type: array
+          collectionFormat: csv
+          items:
+            type: string
+            enum:
+              - isActive
+              - keywords
+              - aspects
+              - subjects
+              - subjectQuery
+              - createdAt
+              - description
+              - helpEmail
+              - helpUrl
+              - id
+              - name
+              - updatedAt
+              - - isActive
+              - - keywords
+              - - aspects
+              - - subjects
+              - - subjectQuery
+              - - createdAt
+              - - description
+              - - helpEmail
+              - - helpUrl
+              - - id
+              - - name
+              - - updatedAt
+        -
+          $ref: "#/parameters/limitParam"
+        -
+          $ref: "#/parameters/offsetParam"
+        -
+          name: description
+          in: query
+          type: string
+          maxLength: 4096
+          description: >
+            A description of the generator.
```

Comment:
Indicate that this a filter, like for /aspects. Similar comment for all further params on this endpoint.